lsp-mode.el

```diff
@@ -196,8 +196,7 @@ Optional arguments:
 (define-minor-mode lsp-mode ""
   nil nil nil
   :lighter (:eval (lsp-mode-line))
-  :group 'lsp-mode
-  (lsp--start))
+  :group 'lsp-mode)
```

Comment:
At some point, it'd probably be good to have lsp-mode do more than just mange the lsp-mode variable, but for now this is causing issues.